server/templates/staff/plagiarism/list.student.html

```diff
@@ -0,0 +1,93 @@
+{% import 'staff/_helpers.html' as helpers %}
+
```

Comment:
Could you add an if block here (or outside) so that the box just doesn't show up if there are no cheating results? Many courses will not be able to use the cheating detection tools.